Monika After Story/game/script-topics.rpy

```diff
@@ -17809,3 +17810,29 @@ label monika_conventions:
     m 3eua "I also hear there's a lot of fun activities like idol dance shows, trivia, and other things to do there."
     m 1eubsa "I would love to go to one with you someday, [mas_get_player_nickname()]~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_cupcake_favorite",
+            category=["monika"],
+            prompt="What's your favorite cupcake flavor?",
+            pool=True,
+            unlocked=False,
+            rules={"no_unlock":None},
+            conditional="mas_seenLabels(['monika_cupcake', 'monika_icecream'], seen_all=True)",
+            action=EV_ACT_UNLOCK
+        )
+    )
+
+label monika_cupcake_favorite:
+    m 1rta "Hmm, I'm not sure I really have one..."
+    m 1hub "I like all sorts of different kinds, so it's hard to choose just one!"
+    m 3ekd "I think I've mentioned before how much I miss Natsuki's cupcakes..."
+    m 3eua "One time she made this really strange mint chocolate chip flavored cupcake...{w=0.3}it had mint flavored frosting with chocolate sprinkles and a chocolate cake base."
+    m 4rksdlb "It was one of the strangest things I've ever tasted ahaha!"
```

Comment:
There's missed comma in this line before "ahaha".

Reply:
this is already merged so put this on the typos issue.